test/pylib/scylla_cluster.py

```diff
@@ -1164,11 +1185,19 @@ async def catching_handler(request) -> aiohttp.web.Response:
                     return aiohttp.web.Response(status=500, text=str(e))
             return catching_handler
 
+        def route_history_wrapper(handler: Callable)-> Callable:
+            @wraps(handler)
+            async def inner_wrapper(request):
+                self.logger.info("[ScyllaClusterManager][%s] %s", asyncio.current_task().get_name(), request.url)
```

Comment:
Can you provide sample output from test.py logs, how it looks like with this change?

Reply:
there is some duplication on the logs (with aiohttp lib). but now I can see the task name and request a connection for better troubleshooting
```
12:16:25.412 INFO>  [18/Apr/2024:11:16:18 +0100] "PUT /cluster/addservers HTTP/1.1" 200 529 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.419 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-166] http://api/cluster/server/4/get_log_filename
12:16:25.419 INFO>  [18/Apr/2024:11:16:25 +0100] "GET /cluster/server/4/get_log_filename HTTP/1.1" 200 230 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.424 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-175] http://api/cluster/server/5/get_log_filename
12:16:25.425 INFO>  [18/Apr/2024:11:16:25 +0100] "GET /cluster/server/5/get_log_filename HTTP/1.1" 200 230 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.425 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-176] http://api/cluster/server/6/get_log_filename
12:16:25.426 INFO>  [18/Apr/2024:11:16:25 +0100] "GET /cluster/server/6/get_log_filename HTTP/1.1" 200 230 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.426 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-177] http://api/cluster/server/7/get_log_filename
12:16:25.426 INFO>  [18/Apr/2024:11:16:25 +0100] "GET /cluster/server/7/get_log_filename HTTP/1.1" 200 230 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.427 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-178] http://api/cluster/server/8/get_log_filename
12:16:25.427 INFO>  [18/Apr/2024:11:16:25 +0100] "GET /cluster/server/8/get_log_filename HTTP/1.1" 200 230 "-" "Python/3.11 aiohttp/3.9.3"
12:16:25.429 INFO> [release/topology.test_repr.1] [ScyllaClusterManager][Task-179] http://api/cluster/server/4/stop_gracefully
```